### PR TITLE
AST-589 - Global heading font setting not working in Gutenberg editor

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Global headings font settings not working in Gutenberg editor 

### Screenshots
- Working in customizer & frontend - https://share.bsf.io/Kou44Ymy
- Not reflecting GB editor - https://share.bsf.io/RBullyNm (inheriting from base typography)
- With fix - https://share.bsf.io/4gunnZDw

### Types of changes
- Bug fix 

### How has this been tested?
- 1. Set global headings font 
- 2. Check with free theme to reproduce the issie
- 3. Need to verify that Global heading font setting applicable to all headings in GB editor

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/core/class-gutenberg-editor-css.php [hunk 1]
```diff
@@ -202,7 +202,7 @@ public static function get_css() {
...
+				'.editor-post-title__block .editor-post-title__input,  .edit-post-visual-editor .block-editor-block-list__block h1, .edit-post-visual-editor .block-editor-block-list__block h2, .edit-post-visual-editor .block-editor-block-list__block h3, .edit-post-visual-editor .block-editor-block-list__block h4, .edit-post-visual-editor .block-editor-block-list__block h5, .edit-post-visual-editor .block-editor-block-list__block h6, .edit-post-visual-editor h1.block-editor-block-list__block, .edit-post-visual-editor h2.block-editor-block-list__block, .edit-post-visual-editor h3.block-editor-block-list__block, .edit-post-visual-editor h4.block-editor-block-list__block, .edit-post-visual-editor h5.block-editor-block-list__block, .edit-post-visual-editor h6.block-editor-block-list__block' => array(
```
Comment (Nikschavan) [inline inc/core/class-gutenberg-editor-css.php:205]: do we need this `block-editor-block-list__block ` in the selectors? will this work if we target only h1-h6?

Comment (imnavanath) [reply inc/core/class-gutenberg-editor-css.php:205]: Yes working if we only target h1-h6. 👍 Updating this code...

Comment (imnavanath) [reply inc/core/class-gutenberg-editor-css.php]: Updated and working as well - https://share.bsf.io/8Luoop78

**Review by Nikschavan — APPROVED** [review]

**Review by premanshup — APPROVED** [review]

**Comment by Nikschavan** [comment]
@imnavanath I think this was merged in #2888 Can you confirm?

**Comment by imnavanath** [comment]
> @imnavanath I think this was merged in #2888 Can you confirm?

Yes right, it's already merged.
Closing this PR as this is already merged in next-release [here](https://github.com/brainstormforce/astra/pull/2888/).

**Pull request closed without merging**